plonky2/src/util/mod.rs

```diff
@@ -58,10 +58,61 @@ pub(crate) fn reverse_bits(n: usize, num_bits: usize) -> usize {
 
 #[cfg(test)]
 mod tests {
-    use plonky2_util::{reverse_index_bits, reverse_index_bits_in_place};
+    use plonky2_util::{log2_ceil, log2_strict, reverse_index_bits, reverse_index_bits_in_place};
 
     use crate::util::reverse_bits;
 
+    #[test]
```

Comment:
nit: I think it would be idiomatic to have the tests in the util crate. (I realize I didn't follow that with `reverse_index_bits`... just didn't notice it as I was moving things around.)

Reply:
Ah! I thought that was weird, but I figured there must be a technical reason for that! I'll move them all to `util`

Reply:
Actually, to move the `reverse_index_bits` tests, I'd have to move `reverse_bits` as well, and that touches more code than I'm comfortable changing in this PR